js/index.js

```diff
@@ -1 +1,17 @@
 // Your code goes here
```

Comment:
See what you can do with `document.getElementsByClassName("btn");` and a `for` loop after changing the classes back to `btn` in the HTML.

Reply:
Aww cool. Ok.